android-sdk/src/main/java/com/mobilecoin/lib/FogSeed.java

```diff
@@ -20,7 +24,7 @@
 import java.util.Objects;
 import kex_rng.KexRng.StoredRng;
 
-class FogSeed implements Serializable {
+class FogSeed implements Serializable, Parcelable {
```

Comment:
Can you add a todo to remove serializable? And can you specify why it can't be done in this PR?

Reply:
I'll add a TODO for this and update the PR with why